### ISSUE TITLE
Production Build Error on Android 

### ISSUE THREAD

**Issue by Quadriphobs1** [tkporter/react-native-sms#62]
When build a production version the app I keep getting this error:
```
Execution failed for task ':react-native-sms:verifyReleaseResources'.

java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  Output:  error: resource android:style/TextAppearance.Material.Widget.Button.Borderless.Colored not found.
 /Path-to-project/node_modules/react-native-sms/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:7: error: resource android:attr/colorError not found.
   /Path-to-project/node_modules/react-native-sms/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:11: error: resource android:attr/colorError not found.
   /Path-to-project/node_modules/react-native-sms/android/build/intermediates/res/merged/release/values-v26/values-v26.xml:15: error: style attribute 'android:attr/keyboardNavigationCluster' not found.
   /Path-to-project/node_modules/react-native-sms/android/build/intermediates/res/merged/release/values/values.xml:226: error: resource android:attr/fontStyle not found.
   /Path-to-project/node_modules/react-native-sms/android/build/intermediates/res/merged/release/values/values.xml:226: error: resource android:attr/font not found.
  /Path-to-project/node_modules/react-native-sms/android/build/intermediates/res/merged/release/values/values.xml:226: error: resource android:attr/fontWeight not found.
  error: failed linking references.
```

and I cant just find any means to comprehend whats happening here. Can anyone help please when I remove this package everything seems to be working fine back

**Comment by nashivan** [comment]
I have the same problem :(

**Comment by joeyscarim** [comment]
what worked for me was updating `node_modules/react-native-sms/android/build.gradle` to have:

```
android {
    compileSdkVersion 28
    buildToolsVersion "28.0.0"

    defaultConfig {
        minSdkVersion 16
        targetSdkVersion 27
        versionCode 1
        versionName "1.0"
    }
    lintOptions {
        abortOnError false
    }
}
```

**Comment by aurelienshz** [comment]
Instead of updating a file inside node_modules, which is not a reliable solution for projects shared with other developers, here's a way to override the problematic build config from the top-level build.gradle file (it's monkey-patching the changes that #56 brings in, from the root project Gradle config):

```
subprojects {
    afterEvaluate {project ->
        if (project.name == 'react-native-sms') {
            println("react-native-sms: Overriding SDK and build tools versions " +
                    "(using compileSdkVersion $rootProject.ext.compileSdkVersion, " +
                    "buildToolsVersion $rootProject.ext.buildToolsVersion)")
            println('Temporary hack, waiting for https://github.com/tkporter/react-native-sms/pull/56')
            android {
                compileSdkVersion rootProject.ext.compileSdkVersion
                buildToolsVersion rootProject.ext.buildToolsVersion
            }
        }
    }
}
```

**Comment by tkporter** [comment]
Hey @aurelienshz, I just merged #56. Does that fix the issue for you without the hack?

**Comment by aurelienshz** [comment]
Hey @tkporter, thanks for the help! Just tried `"react-native-sms": "git+https://github.com/tkporter/react-native-sms.git#4451fcf"`, and without my build.gradle hack, and it seems to be working! Thanks a lot for the help!

**Comment by tkporter** [comment]
Awesome, I just published v1.9.0!
